docs/data/material/getting-started/templates/shared-theme/themePrimitives.ts

```diff
@@ -183,7 +183,7 @@ export const getDesignTokens = (mode: PaletteMode) => {
       },
     },
     typography: {
-      fontFamily: ['"Inter", "sans-serif"'].join(','),
+      fontFamily: ['Inter', 'sans-serif'].join(','),
```

Comment:
This is also incorrect—the issue is with `sans-serif`, not the quotes. See https://github.com/mui/material-ui/issues/44324#issuecomment-2466078965. Generic font family names like `sans-serif` should not be in quotes. We can simply do:

```suggestion
      fontFamily: 'Inter, sans-serif',
```